dapp/styles/globals.css

```diff
@@ -300,4 +300,8 @@ section.light {
   .container {
     max-width: 940px !important;
   }
+}
+
+.justify-content-right {
+  text-align: right;
```

Comment:
Bootstrap already has a `text-right` ([see api](https://getbootstrap.com/docs/4.0/utilities/text/)) you can use